assets/src/edit-story/components/panels/preset/karma/stylePresets.karma.js

```diff
@@ -34,7 +29,7 @@ describe('Panel: Style Presets', () => {
   let frame;
 
   const ADD_BUTTON = 'Add style preset';
-  const APPLY_BUTTON = 'Apply style preset';
+  const EDIT_BUTTON = 'Edit style presets';
 
   const addText = async (extraProps = null) => {
```

Comment:
This one should use the `library.textAdd` button as well (though I know it doesn't work with the position props - but anything testing that something is placed at a certain position like this is not doing things in the same way a user does anyway).

Reply:
Ah, okay, now I understand -- you mean that we should get to the point via user actions where the Text element has different props by actually changing these in the design panel instead of adding it directly?

There are quite a few tests that do that -- create the preconditions for testing a specific part instead of getting there via actions -- should we create an issue for rewriting those, too? Thoughts?

Reply:
Yeah, let's do that for later, it's not a priority now. But a test using some API not available to "real" users is not really testing anything, as it's kind of cheating. But I'm good with postponing that a bit.

Reply:
(I did change it for this PR, btw)